assets/js/components/Dashboard/NewPaMessage/NewPaMessage.tsx

```diff
@@ -45,7 +46,36 @@ const NewPaMessage = () => {
     setVisualText(alertMessage);
   };
 
-  const onImportLocations = () => {};
+  const onImportLocations = (informedEntities: InformedEntity[]) => {
+    const importedSigns = informedEntities.reduce((acc, entity) => {
```

Comment:
**suggestion:** I'm pretty sure this is a [`flatMap`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap)

Reply:
Ahh yes. Thanks!